frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/obc-test.spec.ts

```diff
@@ -0,0 +1,124 @@
+import { expect } from 'chai';
+import {
+  ACCESS_KEY,
+  BOUND,
+  MASK,
+  MINUTE,
+  NO_ANNOTATIONS,
+  NOOBAA_LABEL,
+  NS,
+  OBC_NAME,
+  OBC_STORAGE_CLASS,
+  OBC_STORAGE_CLASS_EXACT,
+  SECRET_KEY,
+} from '../utils/consts';
+import { testDeployment, deployment } from '../mocks/deploymentData';
+import { CreateOBCHandler } from '../views/obcPage';
+import { testName } from '../../../integration-tests-cypress/support/index';
+import { detailsPage } from '../../../integration-tests-cypress/views/details-page';
+import { listPage } from '../../../integration-tests-cypress/views/list-page';
+import { modal } from '../../../integration-tests-cypress/views/modal';
+
+describe('Test Object Bucket Claim resource', () => {
+  let obcHandler;
+  let obcUrl;
+
+  before(() => {
+    cy.login();
+    cy.visit('/');
+    cy.install();
+    obcHandler = new CreateOBCHandler(OBC_NAME, testName, OBC_STORAGE_CLASS);
+    obcHandler.createBucketClaim();
+    cy.exec(`echo '${JSON.stringify(deployment)}' | kubectl create -n ${testName} -f -`);
```

Comment:
Move this to the second it block.

Reply:
Do you mean the first one (as it depends on this code block)? Also, pointing out that after we move this to the first it block, the second it block will be dependant on the first one.

Reply:
My bad I thought you were referring to lines 29-32.